### ISSUE TITLE
Fix sort order in json

### ISSUE THREAD

**Issue by AKWolff037** [konkers/ff7run#69]
Time Mage and Beastmaster have "Magic Counter" sorted outside of the "blue" materia ranges in the json and should be updated so all the blue materia is together.

Magic Counter should be before Sense for Beastmaster
Magic Counter should be before Enemy Away for Time Mage

**Comment by cpankow** [comment]
Since @konkers converts the `json5` to `json` before committing changes (see `README.md`), we can implement a materia sorting function there. My familiarity with javascript is less than my primary languages, so I can't do this immediately. I'll take a look at it later tonight or when I get a free moment.

**Comment by AKWolff037** [comment]
What is `json5` anyway ?

**Comment by konkers** [comment]
I like this approach.  The pre-computation saves 1) needing to do it at runtime and 2) having to catch every place we use the data.  We should probably add a test to confirm that the data is sorted so we don't accidentally mess up the conversion in the future.

Conversion can be done here (https://github.com/konkers/ff7run/blob/master/projects/shared/scripts/convert-data.js#L14) though we'd have to special case materia.

@AKWolff037: json5 is a superset of json that attempts to make it easier to deal with for hand writing.  Includes things like comments, optional "s around object keys, non-decimal constants, and (the reason I'm using it here) multi-line strings.  See https://github.com/json5/json5

**Comment by cpankow** [comment]
Yeah, I planned to implemented the special-case materia sort in `convert-data.js`.
